### ISSUE TITLE
Dependencies on other sites blocking [CODE]

### ISSUE THREAD

**Issue by susanodd** [Signbank/Global-signbank#1306]
There were huge problems just now loading the Search page.

Our code depends on `cdnjs.cloudflare.com` to retrieve packages.

`<script src="https://cdnjs.cloudflare.com/ajax/libs/select2/4.0.5/js/select2.full.js"></script>
`

You could see this in the Network traffic that it was unable to retrieve this.
Thus Signbank fails but it is another site that is failing.

Do we want to install these on our server?

**Comment by Woseseltops** [comment]
The major advantage of using external sources like Cloudflare for these libraries is that it simplifies:
* Upgrading to a newer version: you just increase the number in the template. We rarely upgrade, though, so I don't think this argument should have a lot of weight.
* It simplifies a new installation. This argument too goes away if you either (1) include it in our repo (not sure if allowed, but I think we do it for some other Javascript stuff as well) or (2) add to the Ansible playbooks so it installs automatically.

In other words, both reasons aren't that great, so go ahead improving this situation :)

**Comment by susanodd** [comment]
This seems to be a one-off. Perhaps it coincided with the other world-wide software problems on other sites.
